### DISCUSSION TITLE
[已发布]PCL2更新日志 联网更新版

### DISCUSSION THREAD

**Discussion by Joker2184** [Meloong-Git/PCL#4845] [自定义主页]
![QQ截图20240928134529](https://github.com/user-attachments/assets/560fb968-4ffe-4b4e-a174-d67709c8a498)

使用HomepageBuilder构建生成
预览版
图片来自B站
更新日志来自：B站和Github
部分样式资料引用自 Minecraft 新闻主页
协议使用[CC BY-NC-SA 4.0](https://creativecommons.org/licenses/by-nc-sa/4.0/)

GitHub 仓库：[UpdateHomepage](https://github.com/Joker2184/UpdateHomepage)
反馈：[Issue](https://github.com/Joker2184/UpdateHomepage/issues)
预览加载地址：https://updatehomepage.pages.dev/UpdateHomepage.xaml

**Comment by LTCatt** [comment]
如果稳定了觉得可以上了艾特我一下，然后我再加进列表 :D

**Comment by tangge233** [comment]
![image](https://github.com/user-attachments/assets/66ef5a2c-83f9-427d-9a4e-7164568f1349)

这里可以改成一条线以适配不同界面尺寸……

---

![image](https://github.com/user-attachments/assets/3171a73e-75bc-42d8-ad92-43bebc2342a8)

最后一张卡片的 Margin 值需要调整下……

---

![image](https://github.com/user-attachments/assets/2ded0387-0ae0-485c-98b2-248179915ae0)

图片圆角未生效

**Reply by Joker2184** [reply]
oo

**Reply by Joker2184** [reply]
> ![image](https://private-user-images.githubusercontent.com/50769997/371747178-66ef5a2c-83f9-427d-9a4e-7164568f1349.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjc1MjEyNjAsIm5iZiI6MTcyNzUyMDk2MCwicGF0aCI6Ii81MDc2OTk5Ny8zNzE3NDcxNzgtNjZlZjVhMmMtODNmOS00MjdkLTlhNGUtNzE2NDU2OGYxMzQ5LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA5MjglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwOTI4VDEwNTYwMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWU5OWY5NDJjNDM1ZGNiZGY2MzY1YTA2Y2ZhNjc2ZDY5MzNjY2I4YTgzMGRjOTE0ZTFiNjQyNjJhMjg0YzVmMGImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.XtPutkfnzwIabE8bUXL0YuKX1XGV5uOIr6YnaQg0X-o)
> 
> 这里可以改成一条线以适配不同界面尺寸……
> 
> ![image](https://private-user-images.githubusercontent.com/50769997/371747287-3171a73e-75bc-42d8-ad92-43bebc2342a8.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjc1MjEyNjAsIm5iZiI6MTcyNzUyMDk2MCwicGF0aCI6Ii81MDc2OTk5Ny8zNzE3NDcyODctMzE3MWE3M2UtNzViYy00MmQ4LWFkOTItNDNiZWJjMjM0MmE4LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA5MjglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwOTI4VDEwNTYwMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTY4NjY5NWRmNDE5ZGMyZWQzOGMyZmI0ZGJhN2UzYjQ2OWZlYjAwYjA1ZWY0MWZkYmNkZWJjOTFlNTllZjg1OWImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.SZQFkSSj5I2oaolRT8qQjcUQwcJZXaDgHgenBZZtOhk)
> 
> 最后一张卡片的 Margin 值需要调整下……
> 
> ![image](https://private-user-images.githubusercontent.com/50769997/371747350-2ded0387-0ae0-485c-98b2-248179915ae0.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjc1MjEyNjAsIm5iZiI6MTcyNzUyMDk2MCwicGF0aCI6Ii81MDc2OTk5Ny8zNzE3NDczNTAtMmRlZDAzODctMGFlMC00ODVjLTk4YjItMjQ4MTc5OTE1YWUwLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA5MjglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwOTI4VDEwNTYwMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTU2ZTVlZDZhOTIwNjc5ZDJjODlmNDYzNTQyNTAxYWNmZjI1MmNkNjliNzhmMjJlMzZjZWRmNWEwZWFmYmViYTQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.NuRQn73AO1X50yfoFIYmHFqkHXid5SYdqc03tXg5Kv8)
> 
> 图片圆角未生效

线：借用了下信标的 能用了
圆角修了
最后Margin那个需要做成啥样（？
刷新下缓存应该有新的了

**Comment by Joker2184** [comment]
![QQ截图20240928190512](https://github.com/user-attachments/assets/db2bc9be-e1fa-431f-bcd9-2fe9fa682e4a)
这样吗
@tangge233

**Reply by tangge233** [reply]
到下方卡片的距离为 15，代码里有写

---

![image](https://github.com/user-attachments/assets/e9217266-95ab-4da9-95da-fd5326b3f879)

从新闻主页扣过来了一些不必要的内容……

**Reply by Joker2184** [reply]
> 到下方卡片的距离为 15，代码里有写
> 
> ![image](https://private-user-images.githubusercontent.com/50769997/371761522-e9217266-95ab-4da9-95da-fd5326b3f879.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjc1MzEzNDQsIm5iZiI6MTcyNzUzMTA0NCwicGF0aCI6Ii81MDc2OTk5Ny8zNzE3NjE1MjItZTkyMTcyNjYtOTVhYi00ZGE5LTk1ZGEtZmQ1MzI2YjNmODc5LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA5MjglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwOTI4VDEzNDQwNFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWYxYzVmOTg0YjI4NTNhNzNkMTExNTJkN2ViYjE1NjJlNDBlNzFmZjIyNGVhY2M1NWZkYmQxOWVmNjYzM2EwYzQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.L16JJ6yW7daYooQVawx7_J97SUYmY_Iz70Wm-VxtBEE)
> 
> 从新闻主页扣过来了一些不必要的内容……

（不是扣（
是HomepageBuilder的锅（（（
没找到Style文件在哪导致也没法改（

**Reply by Joker2184** [reply]
改完了看看？

**Reply by tangge233** [reply]
LGTM

**Reply by tangge233** [reply]
看到你更新了一些按钮，这里建议将关于另放卡片（就像新闻主页布局那样），不然按钮布局让人感觉奇怪……

![image](https://github.com/user-attachments/assets/5170eb67-084e-4b4a-9ea8-5fb8fa79b066)

**Reply by Joker2184** [reply]
现在可以通过：https://updatehomepage.pages.dev/UpdateHomepageTest.xaml

获取测试页面
![QQ截图20240929181758](https://github.com/user-attachments/assets/2fa27345-b90e-4377-9627-78bc9e10f6b7)

这样可以吗

**Reply by tangge233** [reply]
LGTM

**Comment by Joker2184** [comment]
圆角问题处理了.jpg

**Comment by tangge233** [comment]
![image](https://github.com/user-attachments/assets/ae43d688-3995-4799-9b4e-017d7b33d387)

跳转错页面了，应该是 `https://github.com/Hex-Dragon/PCL2/issues/new/choose` 而不是 `https://github.com/Light-Beacon/PCL2-NewsHomepage/issues/new/choose`

**Reply by Joker2184** [reply]
凌云FIX了

**Reply by MoYuan-CN** [reply]
- Fix in Joker2184/UpdateHomepage#1

**Comment by LTCatt** [comment]
![image](https://github.com/user-attachments/assets/bdd44025-521c-467a-bb73-776a11525c02)
图片在窄窗口下的适应略有问题

2.8.8 加进去了 :D

**Reply by Joker2184** [reply]
龙猫能不能给一下没文字的图片
正式版的绿色
快照版的蓝色
紧急修复的红色
文字这个...修不了.jpg 干脆不用文字了

**Reply by LTCatt** [reply]
需要 Bahnschrift、思源黑体和阿里妈妈方圆体以正常显示文本。
[PCL2 afd title.zip](https://github.com/user-attachments/files/17201773/PCL2.afd.title.zip)

**Reply by Joker2184** [reply]
收到.jpg

**Comment by z0z0r4** [comment]
这个是真的好

**Comment by Joker2184** [comment]
![QQ截图20241001212923](https://github.com/user-attachments/assets/bcc4a9ae-908f-4a65-adb1-f0b848f865c3)

下个版本尝试使用无文字图片.jpg 

解决不了文字显示不全的问题....
那就解决文字.jpg

**Reply by Joker2184** [reply]
*  首先没技术.json 
*  其次每次更新都要改.jpg
* 魔改了龙猫的PSD文件后
  * 只能塞PCL三个字母上方和下方（ 才能让默认大小下不会出现字被分割的情况

**Comment by Joker2184** [comment]
迁移至OSS
http://pclhomeplazaoss.lingyunawa.top:26994/d/Homepages/Joker2184/UpdateHomepage.xaml

**Reply by MoYuan-CN** [reply]
cc @LTCatt

**Reply by LTCatt** [reply]
将在下个版本生效

**Comment by Joker2184** [comment]
预设活跃橙兑换：
0C30-46AC-4B06-90BC
顺便请龙猫把更新日志 -> 重命名为 新闻速报.23333

**Reply by MoYuan-CN** [reply]
已为改名请求重新配置 Label 为 `等待确认`
cc @LTCatt

**Reply by LTCatt** [reply]
你的解锁码：IAjCNR3Q6N5EBk2p33HOR4NENJFxEW3v
在 更多 → 关于与鸣谢 → 输入解锁码 中使用，谢谢支持～

**Reply by LTCatt** [reply]
已重命名

**Comment by Joker2184** [comment]
最近给托管机玩炸了......

暂时没修好........

所以麻烦龙猫大大在下一次更新前先把链接移除 防止给其他用户带来困扰

我需要时间去新机子上重新去配置一下主页....

下下次再加回来....

**Reply by LTCatt** [reply]
ok，好了再发个评论就行

**Comment by Joker2184** [comment]
还得等等（期末考 Sorry

**Reply by llsgllsg** [reply]
9

**Comment by Joker2184** [comment]
修完了 目前没啥问题了 下个版本可以正常访问了

**Reply by Joker2184** [reply]
@LTCatt

**Reply by LTCatt** [reply]
目前无法访问：https://pclhomeplazaoss.lingyunawa.top:26994/d/Homepages/Joker2184/UpdateHomepage.xaml

**Reply by MoYuan-CN** [reply]
哎 HPoP 这几天在迁移物理机，不知道服务商那边在搞什么，反正是 Down 了快一周了
我今天晚上看看先换个源临时把最新的一次备份跑起来好了 不好意思

**Reply by Joker2184** [reply]
> 哎 HPoP 这几天在迁移物理机，不知道服务商那边在搞什么，反正是 Down 了快一周了 我今天晚上看看先换个源临时把最新的一次备份跑起来好了 不好意思

龙猫我们估计得等四月了 墨渊现在没啥空  托管在我们那的基本都挂了:（
